.gitlab-ci.yml

```diff
@@ -80,6 +80,7 @@ before_script:
       - config/Makefile
       - config/coq_config.py
       - config/coq_config.ml
+      - config/dune.c_flags
```

Comment:
When is this generated and what is this used for?

Reply:
This is always generated by configure, it is used to setup the floating point flags for the kernel float vm machinery.